sqlobject/src/main/java/org/jdbi/v3/sqlobject/customizer/internal/BindListFactory.java

```diff
@@ -49,10 +48,14 @@ public SqlStatementParameterCustomizer createForParameter(Annotation annotation,
                 case NULL:
                     stmt.define(name, "null");
                     return;
+                case JAVA_NULL:
+                    stmt.define(name, null);
+                    return;
                 case THROW:
```

Comment:
Should these behaviors be a dynamic method on the Enum, rather than a switch statement?  It eliminates the `default` case.